field.go

```diff
@@ -708,29 +712,78 @@ func (f *Field) ClearBit(rowID, colID uint64, t *time.Time) (changed bool, err e
 	} else if v {
 		changed = v
 	}
-
-	// Exit early if no timestamp is specified.
-	if t == nil {
+	if len(f.views) == 1 { // assuming no time views
 		return changed, nil
 	}
-
-	// If a timestamp is specified then clear bits across all views for the quantum.
-	for _, subname := range viewsByTime(viewName, *t, f.TimeQuantum()) {
-		view, err := f.CreateViewIfNotExists(subname)
-		if err != nil {
-			return changed, errors.Wrapf(err, "creating view %s", subname)
+	lastLevel := 0
```

Comment:
the names of these three variables made this confusing to read.
some comments would probably help, but also renaming; particularly the two "level" variables since they don't actually mean the same thing.
- `lastLevel` represents a `len(view.name)`: `standard_2018=6, standard_201801=8, standard_20180102=10, standard_2018010203=12`
- `level` represents the date quantum level: `Y=1, YM=2, YMD=3, YMDH=4` or `Y=1, YD=2` or `M=1, MD=2`, etc. 
- `skipBelow` really means "skip all levels above this value"

I would suggest renaming `lastLevel` to `lastViewNameSize` or something like that, and renaming `skipBelow` to `skipAbove` or `skipLevelsAbove` or something a little more clear. I realize if you envision a tree of time quantums with Y at the top, then levels are below, but the logic here has levels increasing numerically.